src/portfolio.jl

```diff
@@ -1,28 +1,37 @@
 const PORTFOLIO_KWARGS =
     Set((:name, :description, :data_source, :run_checks, :unit_portfolio))
 
+const DATA_FORMAT_VERSION = "0.1.0"
+
 const DEFAULT_AGGREGATION = PSY.ACBus
 
-const PORTFOLIO_STRUCT_DESCRIPTOR_FILE = joinpath(
-    dirname(pathof(PowerSystemsInvestmentsPortfolios)),
-    "descriptors",
-    "portfolio_structs.json",
-)
+const PORTFOLIO_STRUCT_DESCRIPTOR_FILE = nothing
 
 mutable struct PortfolioMetadata <: IS.InfrastructureSystemsType
     name::Union{Nothing, String}
     description::Union{Nothing, String}
     data_source::Union{Nothing, String}
 end
 
+mutable struct PortfolioFinancialData <: IS.InfrastructureSystemsType
+    "Base economic year. All costs will be converted to a net present value in this year."
+    base_year::Int64
+    "Discount rate"
+    discount_rate::Float64
+    "Inflation rate"
+    inflation_rate::Float64
+    "Interest rate"
+    interest_rate::Float64
+end
+
 #TODO: Define if we are going to support unit systems
 #TODO: Make immutable
```

Comment:
Remove these TODO. We will not support unit systems, and change it to be immutable and see if it everything works